.azure-pipelines/env.yml

```diff
@@ -181,15 +181,36 @@ jobs:
       set -e
 
       PUBLISH_GITHUB_RELEASE=false
-      PUBLISH_NETLIFY=false
-      if [[ -n "$POSTSUBMIT" && "$NOSYNC" != true ]]; then
-          PUBLISH_NETLIFY=true
-      fi
-      if [[ -n "$POSTSUBMIT" && "$NOSYNC" != true && "$ISSTABLEBRANCH" == True && "$(state.isdev)" == false ]]; then
-          PUBLISH_GITHUB_RELEASE=true
+      PUBLISH_DOCKERHUB=false
+      PUBLISH_DOCS=false
+      PUBLISH_DOCS_LATEST=false
+      PUBLISH_DOCS_RELEASE=false
+
+      if [[ "$ISSTABLEBRANCH" == True && -n "$POSTSUBMIT" && "$NOSYNC" != true ]]; then
+          # Build docs for publishing either latest or a release build
+          PUBLISH_DOCS=true
+          if [[ "$ISMAIN" == True ]]; then
+              # Update the Dockerhub README
+              PUBLISH_DOCKERHUB=true
+              if [[ "$(state.isDev)" == true ]]; then
+                  # Postsubmit on `main` trigger rebuild of latest docs
+                  PUBLISH_DOCS_LATEST=true
+              fi
```

Comment:
So we tagged the main branch to create a new main release like v1.27.0, we will not publish the docs, right?
And until the release branch is created, the docs will published?

Reply:
on `main` commits, netlify rebuilds the website from the current

on releases ci pushes a copy directly the website repo